tests/tests.cpp

```diff
@@ -8,6 +15,7 @@
 #include "catch.hpp"
 
 #include "../data_structures/quadtree.cpp"
+#include "../data_structures/rtree.cpp"
```

Comment:
```suggestion
#include "../data_structures/rtree.hpp"
```
You want to include the header file, not the implementation file, right?